sharktank/sharktank/examples/paged_llm_v1.py

```diff
@@ -213,25 +227,35 @@ def main():
         help="DType to use for activations in the model",
         default="float32",
     )
+    parser.add_argument(
+        "--use-fp8-quantization",
+        help="DType to use for activations in the model",
+        default="false",
```

Comment:
check out optionalBooleanAction or something like that, you don't have to do all the postprocess parsing like line 241